src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -456,7 +570,28 @@ def __call__(
         params: dict = None,
         dropout_rng: PRNGKey = None,
         train: bool = False,
+        output_attentions: Optional[bool] = None,
+        output_hidden_states: Optional[bool] = None,
+        return_dict: Optional[bool] = None,
     ):
+        output_attentions = output_attentions if output_attentions is not None else self.config.output_attentions
+        output_hidden_states = (
+            output_hidden_states if output_hidden_states is not None else self.config.output_hidden_states
+        )
+        return_dict = return_dict if return_dict is not None else self.config.return_dict
+
+        # make sure that model does not return dict when in jitted mode
+        if not isinstance(input_ids, (DeviceArray, np.ndarray)) and return_dict:
+            logger.warn(
+                "Model cannot return dictionary in jitted mode. Setting `return_dict=False`... To suppress this warning, please set ``return_dict=False``."
+            )
+            return_dict = False
+
+        if output_attentions:
+            raise NotImplementedError(
```

Comment:
@sgugger @marcvanzee @avital - For now it is not possible to return attentions because Flax `flax.linen.dot_product_attention` does not allow it. Not sure what the best approach is here - could we maybe adapt `dot_product_attention` slightly to also return attention_weights with a flag?

Reply:
Oh this was something I forgot to do! Let me to it today or tomorrow -- we can refactor `dot_product_attention` to be composed of two functions, one of which just returns the attention mask